modules/installation-network-user-infra.adoc

```diff
@@ -14,7 +14,9 @@ establish a network connection to download their Ignition config files.
 
 It is recommended to use the DHCP server to manage the machines for the cluster
 long-term. Ensure that the DHCP server is configured to provide persistent IP
-addresses and host names to the cluster machines.
+addresses and host names to the cluster machines. DHCP must also configure
+a default DNS search zone. The Kubernetess API server must be able to resolve
```

Comment:
I probably wouldn't state we have a requirement for a default DNS search zone.  The requirement is simply that all machines in the cluster can resolve each other.  In the case where API servers and worker nodes are in different zones then a default DNS search is one way to handle the problem.   Another way is simply having the hosts fully qualified names recorded in their node object.  The problem hit in testing was that Kubernetes didn't know the FQDNs so when the API Servers tried to proxy requests it didn't know how to perform the correct DNS lookup.